catalog/edf-methanesat-ee/projects_edf-methanesat-ee_assets_public-preview_L3concentration.jsonnet

```diff
@@ -0,0 +1,168 @@
+    with the short side that can be stretched wider at higher viewing zenith angles 
+    (which is why some targets appear more rectangular than others). At nadir, the 
+    swath width is ~220 km. The data on these native soundings is Level 2 (L2) data. 
+    The precision of the L2 XCH₄ is 22-35 ppb (corresponding to 0.7-0.2 surface albedo).
+    L2 concentrations (XCH₄) are retrieved from the measured high resolution (0.08 nm 
+    sampling, 0.24 nm resolution) spectra from the 1.6 micron band using the CO2-Proxy 
+    method ([Chan Miller et al. (2024)](https://doi.org/10.5194/amt-17-5429-2024)). 
+    The Level 3 (L3) dataset is created by regridding the L2 data from the native 
+    sounding locations to a regular 45m x 45m grid using the approach detailed in 
+    [Sun et al. (2018)](https://doi.org/10.5194/amt-11-6679-2018). 
+    
+    Contact the data provider for more information about the project at this link: 
+    [https://www.methanesat.org/contact/](https://www.methanesat.org/contact/).
+  |||,
+  keywords: [
+    'climate',
+    'emissions',
+    'ghg',
+    'methane',
+    'edf',
+    'methanesat',
+  ],
+  providers: [
+    ee.producer_provider('Environmental Defense Fund - MethaneSAT', 'https://methanesat.org'),
+    // This is always the last entry.
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2006-01-24T00:00:00Z', '2025-01-17T00:00:00Z'),
+  summaries: {
+    'gsd': [10.2],
+    'eo:bands': [
+      {
+        name: 'XCH4',
+        description: 'Retrieved column-averaged dry-air CH4 mole fraction.',
+        'gee:units': units.ppb,
+      },
+      {
+        name: 'albedo',
+        description: 'Ratio of solar radiance at the ground to observed ' +
+        'radiance at the sensor.',
+      },
+      {
+        name: 'surface_pressure',
+        description: 'Air pressure at the surface.',
+        'gee:units': units.hectopascal,
+      },
+      {
+        name: 'terrain_height',
+        description: 'Elevation with respect to WGS84 reference ellipsoid.',
+        'gee:units': units.kilometer,
+      }
+    ],
+    XCH4: {minimum: 1894.00, maximum: 2114.65, 'gee:estimated_range': true},
+    albedo: {minimum: 0, maximum: 1, 'gee:estimated_range': false},
+    surface_pressure: {minimum: 725.95, maximum: 1011.33, 'gee:estimated_range': true},
+    terrain_height: {minimum: 0.026, maximum: 2.915, 'gee:estimated_range': true},
+    'gee:visualizations': [
+      {
+        display_name: 'Retrieved column-averaged dry-air CH4 mole fraction.',
+        lookat: { lon: -102.9, lat: 32, zoom: 8 },
+        image_visualization: {
+          band_vis: {
+            min: [1870],
+            max: [2030],
+            bands: ['XCH4'],
+            palette: ['navy', 'magenta', 'orange', 'yellow'],
+          }
+        },
+      },
+    ],
+    'gee:schema': [
+      {
+        name: 'flight_id',
+        description: 'Research flight number.',
```

Comment:
Does this still apply to satellite data?

More importantly, I don't see any properties on the current images. (You can specify properties in the manifest when you register COG-backed assets). At the very least, please set the image start and end dates.

Reply:
this was fixed.  Also, added start and end times along with additional properties to the registered assets and captured them in the respective `gee::schema` blocks